src/blocks/blocks/modal/block.json

```diff
@@ -0,0 +1,150 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "themeisle-blocks/modal",
+	"title": "Modal",
+	"category": "themeisle-blocks",
+	"description": "Display your content in beautiful Modal with many customization options. Powered by Otter.",
+	"keywords": [ "modal", "lightbox" ],
+	"textdomain": "otter-blocks",
+	"attributes": {
+		"id": {
+			"type": "string"
+		},
+		"minWidth": {
+			"type": ["number", "string"]
+		},
+		"maxWidth": {
+			"type": ["number", "string"]
+		},
+		"trigger": {
+			"type": "string"
+		},
+		"wait": {
+			"type": "number"
+		},
+		"anchor": {
+			"type": "string"
+		},
+		"scroll": {
+			"type": "number"
+		},
+		"showClose": {
+			"type": "boolean",
+			"default": true
+		},
+		"outsideClose": {
+			"type": "boolean",
+			"default": true
+		},
+		"anchorClose": {
+			"type": "boolean",
+			"default": false
+		},
+		"closeAnchor": {
+			"type": "string"
+		},
+		"recurringClose": {
+			"type": "boolean",
+			"default": false
+		},
+		"recurringTime": {
+			"type": "number"
+		},
+		"backgroundColor": {
+			"type": "string"
+		},
+		"closeColor": {
+			"type": "string"
+		},
+		"overlayColor": {
+			"type": "string"
+		},
+		"overlayOpacity": {
+			"type": "number"
+		},
+		"lockScrolling": {
+			"type": "boolean"
+		},
+		"padding": {
+			"type": "object"
+		},
+		"paddingTablet": {
+			"type": "object"
+		},
+		"paddingMobile": {
+			"type": "object"
+		},
+		"borderWidth": {
+			"type": "object"
+		},
+		"borderRadius": {
+			"type": "object"
+		},
+		"borderColor": {
+			"type": "string"
+		},
+		"borderStyle": {
+			"type": "string"
+		},
+		"width": {
+			"type": "string"
+		},
+		"widthTablet": {
+			"type": "string"
+		},
+		"widthMobile": {
+			"type": "string"
+		},
+		"heightMode": {
+			"type": "string"
+		},
+		"height":{
+			"type": "string"
+		},
+		"heightTablet":{
+			"type": "string"
+		},
+		"heightMobile":{
+			"type": "string"
+		},
+		"verticalPosition": {
+			"type": "string"
+		},
+		"horizontalPosition": {
+			"type": "string"
+		},
+		"verticalPositionTablet": {
+			"type": "string"
+		},
+		"horizontalPositionTablet": {
+			"type": "string"
+		},
+		"verticalPositionMobile": {
+			"type": "string"
+		},
+		"horizontalPositionMobile": {
+			"type": "string"
+		},
+		"closeButtonType": {
+			"type": "string"
+		},
+		"boxShadow": {
+			"type": "object",
+			"default": {
+				"active": false,
+				"colorOpacity": 50,
+				"blur": 5,
+				"spread": 1,
+				"horizontal": 0,
+				"vertical": 0
+			}
+		},
+		"disableOn": {
+			"type": "string"
+		}
+	},
+	"editorStyle": "otter-popup-editor",
+	"style": "file:../popup/style.css",
```

Comment:
@HardeepAsrani, I have a problem. Ideally, we should use the same CSS file for the popup since it is the same.

However, using the same CSS hook handler as the one in the popup does not seem to work. My workaround is just creating another entry, so we still have duplicates. 

Any tips on this?

Reply:
This might be a stupid idea, but have you tried to register the same script twice with a different handle and see if that works? And does that add the same script twice or avoids doing that?